config/routes.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  devise_for :users
+
+  devise_for :users 
```

Comment:
- [ ] I didn't see any API endpoint for authentication, both registration and login
       1.  Login endpoint - `POST : /login`
        2. Registration endpoint - `POST : /register or /signup`

Reply:
Hello mr @mmsesay,
thank you so much for your time , I see that you mention that i didn't put any endpoints for registration and login! 
but i did ... as you can see in the Screenshot 
<img width="536" alt="Screenshot 2022-11-11 220310" src="https://user-images.githubusercontent.com/57154534/201430524-e65fff73-a32a-4852-90c5-6af13dc49bca.png">

Reply:
Hi @TahenyBELGUITH thanks for the reply. 
From the screenshot, only the endpoints to `get` and `create a comment for a specific post` is shown.